CODIGO/General.bas

```diff
@@ -1809,19 +1809,20 @@ min_Err:
 End Function
 
 
-Public Function LoadInterface(FileName As String) As IPicture
+Public Function LoadInterface(FileName As String, Optional localize As Boolean = True) As IPicture
```

Comment:
`byval filename`

Reply:
Why do we need `Optional localize As Boolean = True` ?

Reply:
we do, there are a lot of images like icons and unlocalized text that we are duplicating just for this

Reply:
I don't think it's a good idea, we must add any missing icon or image for english and spanish

Reply:
whats the point of duplicate a bin icon?

Reply:
Ok, I didn't understand what you meant.

I see the point.